cmd/cmd.go

```diff
@@ -587,6 +620,8 @@ func generateInteractive(cmd *cobra.Command, model string, wordWrap bool, format
 		fmt.Fprintln(os.Stderr, "  /set nowordwrap   Disable wordwrap")
 		fmt.Fprintln(os.Stderr, "  /set format json  Enable JSON mode")
 		fmt.Fprintln(os.Stderr, "  /set noformat     Disable formatting")
+		fmt.Fprintln(os.Stderr, "  /set image add <id> <file path>    Add image to chat")
```

Comment:
Might be a silly question, but is `<id>` required?

Reply:
no, this got left in by accident as part of a merge.